clmm/dataops/__init__.py

```diff
@@ -242,12 +186,12 @@ def compute_background_probability(
     """
     if validate_input:
         validate_argument(locals(), "z_lens", float, argmin=0, eqmin=True)
-        validate_argument(locals(), "z_source", "float_array", argmin=0, eqmin=True, none_ok=True)
+        validate_argument(locals(), "z_src", "float_array", argmin=0, eqmin=True, none_ok=True)
 
     if use_pdz is False:
```

Comment:
There is no `use_pdz` flag.

Reply:
I think we can keep this function as it is

Reply:
Yes. Just saw that `use_pdz` was still an argument of the function. It is just not in the docstrings.